src/components/CompareResults/ResultsMain.tsx

```diff
@@ -20,13 +27,46 @@ function ResultsMain() {
     title: style({
       margin: 0,
       marginBottom: Spacing.Medium,
+      fontStyle: 'Heading XS',
+      fontFamily: 'Metropolis',
+      fontSize: '24px',
+      fontWeight: 700,
+      lineHeight: '28px',
```

Comment:
please use FontsRaw.HeadingXS from `src/styles/Fonts.ts` when it makes sense
and of course then add the ones that are different.

(in the future we'll want to use MUI's typography but I think this would be more work atm)